File: module-3 Assignment/Conditional Logic Programs/Control & Conditional statements/Q6.c
#include<stdio.h>
int main(){
	char ch;
	
	printf("Enter an alphabet letter: ");
	scanf("%c",&ch);
	
	if((ch=='a' || ch=='e' || ch=='i' || ch=='o' || ch=='u') || (ch=='A' || ch=='E' || ch=='I' || ch=='O' || ch=='U')){
		printf("%c is vowel",ch);
	}
	else if((ch>='a' && ch<='z') || (ch>='A' && ch<='Z')){
		printf("%c is consonant",ch);
	}
	else{
		printf("%c is not a letter",ch);
	}
	
	return 0;
}

File: Python Assignments/Module-2 Fundamentals of python/Q-6.py
# test whether a passed letter is a vowel or not

letter = input("Enter an alphabet: ").lower() # using lower built in function to transform any letter from upper to lower

# applying conditions to find vowel
if letter == 'a' or letter == 'e' or letter == 'i' or letter == 'o' or letter == 'u' :
    print(f"Letter {letter} is vowel!")
else : # if it is not vowel then else part will execute
    print(f"Letter {letter} is not vowel!")